CHARTER.adoc

```diff
@@ -0,0 +1,195 @@
+|[big]*Open Geospatial Consortium*
+|Submission Date: 2023-05-01
+|Approval Date: 
+|Internal reference number of this OGC(R) document: 23-xxx
+|Category: OGC(R) Standards Working Group Charter
+|Authors: Christophe Noel, Brianna R. Pagán
+|===
+
+[cols = "^", frame = "none"]
+|===
+|[big]*{titletext}*
+|===
+
+[cols = "^", frame = "none", grid = "none"]
+|===
+|*Copyright notice*
+|Copyright (C) 2021 Open Geospatial Consortium
+|To obtain additional rights of use, visit http://www.opengeospatial.org/legal/
+|===
+
+<<<
+
+To: OGC members & interested parties
+
+A new OGC Standards Working Group (SWG) is being formed. The OGC members listed below have proposed the OGC GeoZarr SWG.  The SWG proposal provided in this document meets the requirements of the OGC Technical Committee (TC) Policies and Procedures.
+
+The SWG name, statement of purpose, scope, list of deliverables, audience, and language specified in the proposal will constitute the SWG's official charter. Technical discussions may occur no sooner than the SWG's first meeting.
+
+This SWG will operate under the OGC IPR Policy. The eligibility requirements for becoming a participant in the SWG at the first meeting (see details below) are that:
+
+* You must be an employee of an OGC member organization or an individual
+member of OGC;
+
+* The OGC member must have signed the OGC Membership agreement;
+
+* You must notify the SWG chair of your intent to participate to the first meeting. Members may do so by logging onto the OGC Portal and navigating to the Observer page and clicking on the link for the SWG they wish to join and;
+
+* You must attend meetings of the SWG. The first meeting of this SWG is at the time and date fixed below. Attendance may be by teleconference.
+
+Of course, participants also may join the SWG at any time. The OGC and the SWG welcomes all interested parties.
+
+Non-OGC members who wish to participate may contact us about joining the OGC. In addition, the public may access some of the resources maintained for each SWG: the SWG public description, the SWG Charter, Change Requests, and public comments, which will be linked from the SWG’s page.
+
+Please feel free to forward this announcement to any other appropriate lists. The OGC is an open standards organization; we encourage your feedback.
+
+== Purpose of the Standards Working Group
+
+The GeoZarr Standard Working Group (SWG) is chartered to develop a Zarr encoding for geospatial gridded data in the form of Zarr conventions (based on the approach described in the draft Zarr Enhancement Proposal 4).  Zarr specifies a protocol and format used for storing Zarr arrays, while GeoZarr defines **conventions** and recommendations for storing **multidimensional georeferenced grid** of geospatial observations (including rasters). The GeoZarr SWG will also work on improving the Climate and Forecast (CF) conventions if necessary, particularly for alternative coordinate reference system (CRS) encoding if relevant.
```

Comment:
> The GeoZarr SWG will also work on improving the Climate and Forecast (CF) conventions if necessary, particularly for alternative coordinate reference system (CRS) encoding if relevant.

Not sure we want to officially commit to revising CF conventions. Consider relaxing this phrasing to something like:

> If appropriate, the GeoZarr SWG will **develop proposed changes** to the CF conventions...

Reply:
I agree and would suggest working with other CF groups in developing proposed changes. So perhaps:

> If appropriate, the GeoZarr SWG will, working with the OGC netCDF SWG and the CF community, develop proposed changes to the CF conventions ...

The OGC netCDF SWG is responsible for the OGC CF and netCDF documents. The [CF community governance process](https://cfconventions.org/governance.html) is how changes and enhancements are made to the CF specification.

Reply:
I agree also

Reply:
```suggestion
The GeoZarr Standard Working Group (SWG) is chartered to develop a Zarr encoding for geospatial gridded data in the form of Zarr conventions (based on the approach described in the draft Zarr Enhancement Proposal 4).  Zarr specifies a protocol and format used for storing Zarr arrays, while GeoZarr defines **conventions** and recommendations for storing **multidimensional georeferenced grid** of geospatial observations (including rasters). If appropriate, the GeoZarr SWG may also contribute to improving the Climate and Forecast (CF) conventions (e.g. alternative CRS encoding), then propose changes following the [CF community governance process](https://cfconventions.org/governance.html).
```